src/pages/Opportunities.jsx

```diff
@@ -0,0 +1,229 @@
+            <h2 className="text-2xl font-extrabold">
+              482 volunteers are needed
+            </h2>
+            <span className="text-slate-500">Showing 1-10</span>
+          </div>
+
+          {opportunitiesList.map((el, index) => (
+            <div className="flex mb-8 border-b pb-8" key={index}>
+              <div className="mr-4">
+                <svg
+                  fill="none"
+                  stroke="currentColor"
+                  viewBox="0 0 200 200"
+                  preserveAspectRatio="none"
+                  aria-hidden="true"
+                  className="border w-32 text-gray-300"
+                >
+                  <path
+                    d="M0 0l200 200M0 200L200 0"
+                    strokeWidth="1"
+                    vector-effect="non-scaling-stroke"
+                  ></path>
+                </svg>
+              </div>
+              <div>
+                <h4>
+                  <a
+                    href={`/opportunities/${el.id}`}
+                    className="underline text-lg text-blue-500 hover:text-blue-400"
+                  >
+                    {el.title}
+                  </a>
+                </h4>
+                <div className="text-sm text-slate-500">
+                  <span>{el.date}, </span>
+                  <span>{el.location}</span>
+                </div>
+                <p className="mb-2">{el.description}</p>
+                <button className="font-medium rounded-full text-sm px-5 py-2.5 text-center bg-slate-200">
+                  Details
+                </button>
+              </div>
+            </div>
+          ))}
+
+          <div className="flex items-center justify-between">
+            <div className="flex flex-1 justify-between sm:hidden">
+              <a
+                href="#"
+                className="relative inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50"
+              >
+                Previous
+              </a>
+              <a
+                href="#"
+                className="relative ml-3 inline-flex items-center rounded-md border border-gray-300 bg-white px-4 py-2 text-sm font-medium text-gray-700 hover:bg-gray-50"
+              >
+                Next
+              </a>
+            </div>
+            <div className="hidden sm:flex sm:flex-1 sm:items-center sm:justify-center">
+              <nav
+                aria-label="Pagination"
+                className="isolate inline-flex -space-x-px"
+              >
+                <a
+                  href="#"
+                  className="relative inline-flex items-center rounded-l-md px-2 py-2 text-gray-400 hover:bg-gray-50 focus:z-20 focus:outline-offset-0"
+                >
+                  <span className="sr-only">Previous</span>
+                  <ChevronLeftIcon aria-hidden="true" className="h-5 w-5" />
+                </a>
+                <a
```

Comment:
Quite a bit of repetitive code here - think about how you can simplify. E.g., if you have a variable for number of pages somewhere, can you map over an array to generate?

Reply:
@justgogame